### ISSUE TITLE
No more provinces with new caches

### ISSUE THREAD

**Issue by harrieklomp** [opencaching/opencaching-pl#2064]
From today or yesterday evening provinces are not shown anymore with new caches.  This is on all nodes

![Selectie_350](https://user-images.githubusercontent.com/1060518/58903410-a7202a00-8705-11e9-8033-9c75fb974f22.png)

**Comment by kojoty** [comment]
same situation is probably on cache edition

**Comment by harrieklomp** [comment]
With cache edition this should be solved very soon.
With every edit of a cache the region is gone :-(

**Comment by kojoty** [comment]
as a workaround try: /sys/fixCachesLocation
please note: every refresh takes 100 records with broken location so if you see anything refresh again - if you got 0/0/0 it means all locations are fixed 

you can do it from time to time until I fix it - I'm going to fix it but not expect it soon... (I will be on vacation next 4 weeks...)

**Comment by harrieklomp** [comment]
I do not have much work on OCNL. It is more that OCPL is a bit behind the facts. Please give me "sysAdmin" rules to OCPL so i can run /sys/fixCachesLocation from time to time in those 4 weeks :-)

**Comment by andrixnet** [comment]
Not sure if related or a completely different issue. 
I just created a new listing on OCRO (production, which is in sync). 
On cache creation, region was detected and set to "Bucuresti-Ilfov".
Viewing the cache shows it with the correct region.
Editing the cache has the following 2 behaviours: 
- regions dropdown is populated but "all regions" is shown as currently selected
- any slight change in coordinates does not update region dropdown (as new cache page)
- I manually set the region, save cache. Open edit cache again, same problem.
